target/linux/ath79/image/generic.mk

```diff
@@ -441,6 +441,21 @@ define Device/comfast_cf-wr752ac-v1
 endef
 TARGET_DEVICES += comfast_cf-wr752ac-v1
 
+define Device/compex_wpj344-16m
+  SOC := ar9344
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2
```

Comment:
usb-core is selected by usb2, and can be dropped here.

Reply:
done - pushing in a moment